### ISSUE TITLE
Result on IMDB

### ISSUE THREAD

**Issue by digbangbang** [k1l1/CoCoFL#1]
Hi, good works! I wonder how to reproduce ur result on IMDB, when I use ur code, the acc will always keep in 0.5, here is my executing code

`python3 main.py --algorithm FedAvg --network Transformer --dataset IMDB --data_distribution IID --seed 10 --n_rounds 1000 --lr 0.1 --lr_schedule 600 800 --torch_device cuda:1`

Is any problem I've made? plss reply me, thank u!!

**Comment by digbangbang** [comment]
OK, I got the acc higher when training a long time, it is strange when in the begining, the acc kept in 0.5, maybe there is a warmup. Thanks ur wonderful job!

**Comment by k1l1** [comment]
Hi,

Thank you for your interest in this work and for bringing up this issue.

I reviewed the GitHub code and discovered that the automatically downloaded IMDB data does not match the split used in the paper. To address this, I have manually uploaded the correct IMDB data to the repository.

The automatically downloaded data uses a 50%/50% test/train split, whereas the paper uses a 20%/80% split. This update should help you fully reproduce the results.

Best,
Kilian
